### PR TITLE
gui: Translate fancytree messages in Versions modal

### PR DESCRIPTION
**Pull request by tomasz1986**
Currently, the default fancytree info/error messages are used. This
means that a) they are English-only, and b) they are very generic. With
this commit, the messages are added to the translatable strings, and
they are also more specifically related to file versioning.

On a side note, the "moreData" string has been left out on purpose, as
it is not used in the current code. It can be added later if needed.

Signed-off-by: Tomasz Wilczyński <twilczynski@naver.com>

#### Before

image

#### After

image

### CODE REVIEW

File: gui/default/assets/lang/lang-en.json [hunk 1]
```diff
@@ -128,6 +128,7 @@
    "Error": "Error",
    "External File Versioning": "External File Versioning",
    "Failed Items": "Failed Items",
+   "Failed to load file versions.": "Failed to load file versions.",
```
Comment (acolomb) [inline gui/default/assets/lang/lang-en.json:131]: These seem to not get picked up automatically by `go run build.go translate`.  Could you check why?

File: gui/default/syncthing/core/syncthingController.js [hunk 2]
```diff
@@ -2530,6 +2530,11 @@ angular.module('syncthing.core')
                                 table: {
                                     indentation: 24,
                                 },
+                                strings: {
+                                  loading: $translate.instant("Loading..."),
```
Comment (acolomb) [inline gui/default/syncthing/core/syncthingController.js:2534]: Please use consistent indentation of four spaces per level.

**Review by acolomb — CHANGES_REQUESTED** [review]

File: gui/default/assets/lang/lang-en.json (hunk 1 above)
Comment (acolomb) [reply gui/default/assets/lang/lang-en.json:131]: Never mind. They are picked up. My bad.

File: gui/default/syncthing/core/syncthingController.js (hunk 2 above)
Comment (tomasz1986) [reply gui/default/syncthing/core/syncthingController.js]: Fixed. That was my sloppiness caused by a bad copy & paste and relying on the code editor to fix the indentation (which it did wrongly).

**Review by acolomb — APPROVED** [review]
Nice addition!

**Pull request merged**